coalib/processes/BearRunning.py

```diff
@@ -160,9 +160,9 @@ def get_local_dependency_results(local_result_list, bear_instance):
 def run_local_bear(message_queue,
                    timeout,
                    local_result_list,
-                   file_dict,
```

Comment:
To be backwards compatible, or at least break cleanly, `file_dict` here should be replaced with `file_contents`, and the function should check that `file_contents` is of the new expected type.